cli/installation/index.md

```diff
@@ -126,22 +134,27 @@ scoop update keboola/keboola-cli
 
 ### Download
 
-Use an [msi installer](https://cli-dist.keboola.com/?prefix=msi/) or a [precompiled binary](https://cli-dist.keboola.com/?prefix=zip/). 
-You can also install from the source.
-
-## Installing from Source
+Use an [msi installer](https://cli-dist.keboola.com/?prefix=msi/) or a [precompiled binary](https://cli-dist.keboola.com/?prefix=zip/).
+## Build From Source
```

Comment:
`Installing` -> ` build`, myslim, ze je to viac vystiznesie ...

Pod `build` sa vacsinou chape proces kompilacie.
A pod `install` proces presunutia skompilovanej binarky napr. do `/usr/local/bin`.